app/models/training/content.rb

```diff
@@ -16,6 +16,16 @@ def parent
       @parent ||= Training::Module.by_content_id(id)
     end
 
+    # @return [Boolean]
+    def updated_content?
+      created_at < updated_at
+    end
+
+    # @return [Date]
+    def published_at_date
```

Comment:
Imagine the scenario where the Content Editor is working on a draft module, like module 8, previewing it in staging. It isn't published at this point.